site/source/docs/porting/simd.rst

```diff
@@ -1015,3 +1015,154 @@ The following table highlights the availability and expected performance of diff
      - 💣 emulated with complex SIMD+scalar sequence
 
 Only the 128-bit wide instructions from AVX instruction set are available. 256-bit wide AVX instructions are not provided.
+
+
+====================================================== 
+Compiling SIMD code targeting ARM NEON instruction set
+======================================================
+
+Emscripten supports compiling existing codebases that use ARM NEON by
+passing the `-mfpu=neon` directive to the compiler, and including the
+header `<arm_neon.h>`.
+
+The following table highlights the availability of various intrinsics. 
+
+For detailed information on each intrinsic function, refer to `NEON Intrinsics Reference
+<https://developer.arm.com/architectures/instruction-sets/simd-isas/neon/intrinsics>`_.
+
+
+Function groups (Note there may be intrinsics missing from this list,
+that means they are not implemented):
```

Comment:
It might be good to have a note about contributions for new intrinsics being welcome upstream at SIMDe as well.

Reply:
Oh, and this would be a good place to mention the SIMDe updating script as well, perhaps in a sidebar or note box of some sort. In general we could document the contribution process here.